pallets/gear/src/weights.rs

```diff
@@ -2743,14 +2754,24 @@ impl WeightInfo for () {
             .saturating_add(Weight::from_parts(85_900_587, 0).saturating_mul(r.into()))
     }
     /// The range of component `r` is `[0, 20]`.
+    fn gr_signal_code(r: u32, ) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0`
+        //  Estimated: `0`
+        // Minimum execution time: 96_018_000 picoseconds.
+        Weight::from_parts(100_345_503, 0)
+            // Standard Error: 16_521
+            .saturating_add(Weight::from_parts(2_852_293, 0).saturating_mul(r.into()))
+    }
+    /// The range of component `r` is `[0, 20]`.
     fn gr_signal_from(r: u32, ) -> Weight {
         // Proof Size summary in bytes:
         //  Measured:  `0`
         //  Estimated: `0`
-        // Minimum execution time: 93_761_000 picoseconds.
-        Weight::from_parts(92_637_129, 0)
-            // Standard Error: 309_799
-            .saturating_add(Weight::from_parts(85_725_727, 0).saturating_mul(r.into()))
+        // Minimum execution time: 94_600_000 picoseconds.
+        Weight::from_parts(99_974_526, 0)
+            // Standard Error: 22_071
+            .saturating_add(Weight::from_parts(2_948_468, 0).saturating_mul(r.into()))
```

Comment:
It couldn't be like that
Its pretty similar to `gr_reply_to`

Reply:
I think I found the reason. I've placed the function body in `handle()` instead of `handle_signal()`. After fixing it and verifying on my machine, everything is fine, and the `r` component weights are back to somewhat normal. 

Rerunning benchmarks now; I will comment again when it's done.

Reply:
Yes, now the weights are back to normal